scripts/generate_conda_file.sh

```diff
@@ -108,7 +108,8 @@ ${pyspark}- pyarrow>=0.8.0
 ${gpu}  - numba>=0.38.1 
   - gitpython>=2.1.8
   - pydocumentdb>=2.3.3
-  - azureml-core>=0.1.74
+${gpu}  - nvidia-ml-py3>=7.352.0
+  - dataclasses>=0.6
```

Comment:
a question, are you considering that our user will use fastai in a non-gpu environment? As of now, all TF code is considered to go on a gpu env

Reply:
Fast ai should run without a GPU, but it will be slower (the same should be true for TF). I wonder if we want to narrow down the audience for these algorithms to just users that have a GPU. Often times, I would like to try out a github repo on my Mac, just to see how/if it works. What are your thoughts?

Reply:
on a related note: do we have a way to run tests on GPU hardware in automation?

Reply:
for question 1, if I'm doing deep learning I always use a gpu machine, I don't know if I'm the typical user though.
for q2, we have a test pipeline on gpu (in fact our test machine is a gpu machine)

Reply:
ok -- I will put fastai behind the ${gpu} tag then

Reply:
looking at the files, we are installing the non-gpu version of tensorflow if the user does not select gpu